server/public/layout.css

```diff
@@ -24,6 +24,15 @@ body {
   margin: 3rem;
 }
 
+.card-body {
+  line-height: 1rem;
+}
+
+.fa-heart {
+  font-size: 4rem;
+  color: #ee8a8a;
+}
```

Comment:
We don't style external CSS classes like this. It's mentioned in effective-css under code organization.

https://lfz-lms.herokuapp.com/code-guides/Learning-Fuze/curriculum/effective-css